spec-0007/index.md

```diff
@@ -79,7 +80,86 @@ The following users will be affected:
 
 ### Code
 
-TODO: Add example `check_random_state` implementation.
+For example, SciPy may implement this with the following decorator to support both names.
+This is implemented using:
+1. A `check_random_state` function which normalizes the either old or new input to a `Generator` or `RandomState` object.
+   This function only gives a `FutureWarning` when users (probably) NumPy's global legacy `RandomState`.  As noted in point 1. above.
+2. A decorator to deal with the rename changes.  In future versions this will deprecate the old keyword.  At this time, it will ensure that the documentation and auto-completion only shows the new parameter name.
```

Comment:
```suggestion
2. A decorator to deal with the `seed` to `rng` keyword rename.  In future versions, this will deprecate the `seed` keyword. Meanwhile, it ensures that the documentation and auto-completion only advertises the new parameter name.
```